### PR TITLE
Beyond Scale: the Diversity Coefficient as a Data Quality Metric Demonstrates LLMs are Pre-trained on Formally Diverse Data

### LINKED ISSUE #209: Beyond Scale: the Diversity Coefficient as a Data Quality Metric Demonstrates LLMs are Pre-trained on Formally Diverse Data
- Paper name: Beyond Scale: the Diversity Coefficient as a Data Quality Metric Demonstrates LLMs are Pre-trained on Formally Diverse Data
- ArXiv Link: https://arxiv.org/abs/2306.13840

To close this issue open a PR with a paper report using the provided [report template](https://github.com/Dahoas/QDSyntheticData/blob/main/papers/paper_example.md).

### PR DESCRIPTION
**Pull request by haileyschoelkopf**
closes #209

### CODE REVIEW

File: papers/2306.13840-beyond-scale-diversity-coefficient.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Introduces a new metric for measuring (general text) dataset diversity. Evaluates this metric and shows it has a number of intuition-following properties and demonstrates datasets such as the Pile are diverse, and proposes it as a means to measure dataset quality.
+
+## Relevance to survey topic (1-5)
+
+Relevance: 3
+
+## Algorithms
+
+- Diversity Coefficient (metric): generate gradients from GPT-2 on a batch, compute Fisher Information Matrix, take diagonal, compute average cosine similarity between these embeddings
```
Comment (Dahoas) [inline papers/2306.13840-beyond-scale-diversity-coefficient.md:13]: I'm wondering if you can give some intuition about why this is a good diversity measure? Somehow a dataset with batches whose gradients are less correlated is more diverse?
